toolkit/tools/graphpkgfetcher/graphpkgfetcher.go

```diff
@@ -474,25 +474,48 @@ func resolveSingleNode(cloner *rpmrepocloner.RpmRepoCloner, node *pkggraph.PkgNo
 	}
 
 	preBuilt := false
+	// We will try to clone each package. We need at least one to succeed.
+	foundAtLeastOneSource := false
+	cloneErr := error(nil)
```

Comment:
Why are you defining `cloneErr` here? It doesn't seem to be used outside of the loop.